UiPathTeam.Extensions.Tests/DictionaryUnitTests.cs

```diff
@@ -99,6 +99,77 @@ public void TestDictionaryContainsValue()
 
             Assert.IsTrue(boolResult);
         }
+		[TestMethod]
+        public void TestDictionaryContainsKey_KeyExists()
```

Comment:
this unit test is incorrect. you added "key1" to the dictionary and are testing to see if "Key1" exists, which is false.

Reply:
Test dictionary contains value is Cathleen's function, mine is Test Dictionary Contains Key

Reply:
this is for the unit test TestDictionaryContainsKey_KeyExists